test-utils/loadtester/src/remote_node.rs

```diff
@@ -186,94 +205,66 @@ impl RemoteNode {
         transaction: SignedTransaction,
     ) -> Result<String, Box<dyn std::error::Error>> {
         let bytes = transaction.try_to_vec().unwrap();
-        let url = format!("{}{}", self.url, "/broadcast_tx_sync");
-        let result: serde_json::Value = self
-            .sync_client
-            .post(url.as_str())
-            .form(&[("tx", format!("0x{}", hex::encode(&bytes)))])
-            .send()?
-            .json()?;
-        Ok(result["result"]["hash"].as_str().ok_or(VALUE_NOT_STR_ERR)?.to_owned())
+        let params = (to_base64(&bytes),);
+        let message = Message::request(
+            "broadcast_tx_async".to_string(),
```

Comment:
this suppose to be broadcast_tx_commit

Reply:
The `add_transaction`(sync) was meaning using sync client to add "async" txn and later checked by `transaction_committed`. I guess this was more effective than wait until each txn committed, when deploy contracts to a lot of account? @nearmax